lms/lti.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass
+from typing import Callable, Optional
+
+
+@dataclass
+class _ParamMapping:
```

Comment:
This class might be a bit of an overkill but it looks nicer than just plain tuples and if only as a documentation mechanism I think is worth it.